recipes/upgrade.rb

```diff
@@ -66,4 +66,10 @@
     notifies :restart, "service[abiquo-tomcat]" if node['abiquo']['profile'] == 'monolithic' || node['abiquo']['profile'] == 'server'
 end
 
+execute "watchtower-liquibase-update" do
+  only_if { node['abiquo']['profile'] == 'monitoring' }
+  command "/usr/bin/abiquo-watchtower-liquibase update"
+  action :nothing
```

Comment:
This does nothing by default. You need to add a "subscribes" for the emmett or the delorean package (for the one that provides the watchtower DB, I don't know which one it is), so the "run" action is triggered every time the package changes.